tests/functional/v2/server/testListMetrics.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable func-names */
+const assert = require('assert');
+const needle = require('needle');
+const uuid = require('uuid');
+
+const { client: warp10 } = require('../../../../libV2/warp10');
+const { convertTimestamp } = require('../../../../libV2/utils');
+const { operationToResponse } = require('../../../../libV2/constants');
+
+const { generateCustomEvents } = require('../../../utils/v2Data');
+
+const _now = Math.floor(new Date().getTime() / 1000);
+const getTs = delta => convertTimestamp(_now + delta);
+
+const metricResponseKeys = {
+    buckets: 'bucketName',
+    accounts: 'accountId',
+    users: 'userId',
+    service: 'serviceName',
+};
+
+const emptyOperationsResponse = Object.values(operationToResponse)
+    .reduce((prev, key) => {
+        prev[key] = 0;
+        return prev;
+    }, {});
+
+async function listMetrics(level, resources, start, end) {
+    const body = {
+        timeRange: [start, end],
+    };
+    body[level] = resources;
+
+    return needle(
+        'post',
+        `http://localhost:8100/${level}`,
+        body,
+        { json: true },
+    );
+}
+
```

Comment:
extra lines(not sure if your linter catches it)

Reply:
Technically 2 empty lines aren't an error, but consistency is better :)

Reply:
You code is too clean, I couldn't resist.
I majorly wanted to raise the 2 deleteBucketReplication thing :)